examples/tanstack-start/app.config.ts

```diff
@@ -0,0 +1,22 @@
+import { lingui } from "@lingui/vite-plugin";
+import { defineConfig } from '@tanstack/react-start/config'
+import tsConfigPaths from 'vite-tsconfig-paths'
+
+export default defineConfig({
+  tsr: {
+    appDirectory: 'src',
+  },
+  react: {
+		babel: {
+			plugins: ["@lingui/babel-plugin-lingui-macro"],
+		},
+	},
```

Comment:
weird formatting here and in other places, did you apply prettier to these files?

Reply:
prettier is ignoring examples. I forced it to run on this example now